### PR TITLE
[Query Copilot] Sample Prompt implementation and other

### PR DESCRIPTION
**Pull request by Klepic95**
This pull request contains following implementations:

- Sample Prompt Implementation
- Suggested Prompt Implementation
- History Implementation
- Copy code / Delete code button implementation
- Auto show/hide feedback bar
- Query Copilot logo update
- Component tests

[Preview this branch](https://cosmos-explorer-preview.azurewebsites.net/pull/1489?feature.enableCopilot=true)

### CODE REVIEW

**Comment by victor-meng** [comment]
I think there's too much extra spacing in the modal:
image

**Comment by Klepic95** [comment]
> I think there's too much extra spacing in the modal: image
> 
> This is what it looks like in the mockup: image

Fixed with the latest commit :)

**Review by v-darkora — APPROVED** [review]

**Review by v-darkora — DISMISSED** [review]

File: src/Explorer/QueryCopilot/Popup/DeletePopup.tsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    setShowDeletePopup(false);
+  };
+
+  return (
+    <Modal isOpen={showDeletePopup}>
+      <Stack style={{ padding: "16px 24px", height: "auto" }}>
+        <Text style={{ height: 24, fontSize: "18px" }}>
+          <b>Delete code?</b>
+        </Text>
+        <Text style={{ marginTop: 10, display: "inline" }}>
```
[the remark this answers is not in the document]
Comment (Klepic95) [reply src/Explorer/QueryCopilot/Popup/DeletePopup.tsx]: Removed in previous commit :)

File: src/Explorer/QueryCopilot/Popup/DeletePopup.tsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+          This will clear the query from the query builder pane along with all comments and also reset the prompt pane
+        </Text>
+        <Stack style={{ display: "inline-block", marginTop: 50 }}>
+          <PrimaryButton
+            style={{ padding: "10px 20px", width: "fit-content", height: 24, background: "#0078D4" }}
```
[the remark this answers is not in the document]
Comment (Klepic95) [reply src/Explorer/QueryCopilot/Popup/DeletePopup.tsx]: Removed :)

File: src/Explorer/QueryCopilot/QueryCopilotTab.tsx [hunk 3]
```diff
@@ -199,57 +221,70 @@ export const QueryCopilotTab: React.FC<QueryCopilotTabProps> = ({
...
+              setLikeQuery(false);
               setShowCallout(false);
-              submitFeedback({ generatedQuery, likeQuery, description: "", userPrompt: userInput });
+              useQueryCopilot.getState().openFeedbackModal(generatedQuery, false, userInput);
             }}
-            directionalHint={DirectionalHint.topCenter}
+          />
+          <Separator vertical style={{ color: "#EDEBE9" }} />
+          <CommandBarButton
+            onClick={() => navigator.clipboard.writeText(query)}
```
[the remark this answers is not in the document]
Comment (Klepic95) [reply src/Explorer/QueryCopilot/QueryCopilotTab.tsx]: I was not aware of this. Thank you, I will fix it.

**Review by victor-meng — APPROVED** [review]

**Review by v-darkora — APPROVED** [review]

**Pull request merged**